spec/controllers/users_controller_spec.rb

```diff
@@ -100,6 +100,11 @@
               put :update, params: { id: user.to_param, user: { name: 'Trung Le', email: 'newmail@example.com' } }
             }.to change { ActionMailer::Base.deliveries.count }.by(1)
           end
+
+          it "allows updating the requested user" do
```

Comment:
I think it would make sense to have a dedicated test with a user **that is not confirmed yet**.

Reply:
I'm not sure I understood, but given that this is in the context of an unconfirmed user, this is what it does, right?